src/Symfony/Component/Form/Flow/Type/FlowButtonType.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Flow\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Flow\FlowButtonTypeInterface;
+use Symfony\Component\Form\Flow\FlowCursor;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * A submit button with a callable handler for a form flow.
+ *
+ * @author Yonel Ceruto <open@yceruto.dev>
+ */
+class FlowButtonType extends AbstractType implements FlowButtonTypeInterface
```

Comment:
naming, would this make sense?
```suggestion
class ButtonFlowType extends AbstractType implements ButtonFlowTypeInterface
```

Reply:
I'm not sure about this. To me they mean different things:

* `ButtonFlow` = a flow composed of buttons
* `FlowButton` = a button that belongs to a (form) flow

Reply:
The suggested new names makes FlowType a generic suffix for Flow related types, translating to "it's a Button Type from the Flow area". 
They are indeed less meaningful from a reader POV but might be more discoverable. 
That's worth challenging, I've no strong opinion though I do like names that read as a sentence (as it was before) especially for interfaces.

Reply:
both options are acceptable to me, I don't have a strong opinion either way, so I'm fine with whatever we agree on

Reply:
I would stick with the current naming as that IMO is more clear about what it means.